### ISSUE TITLE
UnhandledPromiseRejectionWarning - You are being rate limited.

### ISSUE THREAD

**Issue by mikeytown2** [JoeBanks13/webhook-discord#14]
```
(node:13268) UnhandledPromiseRejectionWarning: {
  "global": false,
  "message": "You are being rate limited.",
  "retry_after": 1739
}

    at emitWarning (internal/process/promises.js:67:17)
    at emitPendingUnhandledRejections (internal/process/promises.js:109:11)
    at process._tickCallback (internal/process/next_tick.js:189:7)
(node:13268) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 2)
    at emitWarning (internal/process/promises.js:75:21)
    at emitPendingUnhandledRejections (internal/process/promises.js:109:11)
    at process._tickCallback (internal/process/next_tick.js:189:7)
(node:13268) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
    at emitWarning (internal/process/promises.js:92:15)
    at emitPendingUnhandledRejections (internal/process/promises.js:109:11)
    at process._tickCallback (internal/process/next_tick.js:189:7)
```

Maybe queue the messages and try again after the given timer?

**Comment by mikeytown2** [comment]
It's for a server monitor I have done and when things go really wrong this message happens.

**Comment by JoeBanks13** [comment]
Ahh okay, that is a valid use case. I'll try get a queue in place.

**Comment by JoeBanks13** [comment]
Fixed in 2.3.0 (now on NPM). Thank you for bringing this up.

**Comment by mikeytown2** [comment]
Awesome!
